modules/optimization/examples/bimaterial_mesh/main.i

```diff
@@ -0,0 +1,117 @@
+# This performs the same 2 parameter inversion problem done
```

Comment:
Please correct me if I am wrong: 

I suppose it would be more appropriate to describe this as a "one parameter inversion problem in a bi-material domain", since the parameter we are solving is only the diffusivity, which is a piecewise constant parameter. This way we keep the consistency with the implementation in `ParameterMeshOptimization`

Reply:
I think there are still 2 parameters in this problem because there are two variables we are controlling during optimization.  I will change the word.